### ISSUE TITLE
accessories and inventorio incombatablity

### ISSUE THREAD

**Issue by Danmccool** [RubixDev/Inventorio#378]
I cant open my inventory with accessories and inventorio enabled at the same time
@RubixDev

**Comment by Danmccool** [comment]
@RubixDev

**Comment by pkej** [comment]
Just for anyone searching, you will notice this error when "I press E but nothing happens". Disabling accessories (and any compatibility mods for accessories) will solve this.

**Comment by SaltSouls** [comment]
Yeah, I am also having this issue. It's kinda unfortunate that this is the case, as I think both these mods would work really well together.

**Comment by Danmccool** [comment]
i closed this because i don't think this will get fixed
